src/views/blocks/parts/details.php

```diff
@@ -131,8 +142,8 @@
 		// Event Cost
 		if ( ! empty( $cost ) ) : ?>
 
-            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ) ?> </dt>
-            <dd class="tribe-events-event-cost"> <?php esc_html_e( $cost ); ?> </dd>
+            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ); ?> </dt>
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 3f1bef08e5c14f3b87963e750bc36aa6</sub>